doc/pfc_watchdog/Pfc_Storm_with_Shared_Headroom_test_plan.md

```diff
@@ -0,0 +1,31 @@
+# PFC Storm with Shared Headroom Test Plan
+
+## Motivation
+
+When a PFC Storm is seen, PFC WatchDog on the Melanox Devices applies Zero Buffer Profiles on Ingress PG and Egress Queues. However if the buffer occupany has already crossed Shared Headroom (i.e. Shared Buffer + Xon) when a PFC Storm is detected, a bug was recently found on mellanox platform. 

```

Comment:
Not sure whether or not necessary to say `a bug was found on Mellanox platform` ...

Reply:
Will remove